src/targets/gpu/fuse_ops.cpp

```diff
@@ -45,8 +45,9 @@ struct fusion
         return result;
     }
 
+    fusion() {}
```

Comment:
Since we are adding a default constructor, we might need to add a `assert(fp)` at the beginning of each method.

Reply:
fixed.